backend/app/views/spree/admin/properties/edit.html.erb

```diff
@@ -1,9 +1,6 @@
 <% content_for :page_title do %>
-  <%= Spree.t(:editing_resource, resource: Spree::Property.model_name.human) %>
-<% end %>
-
-<% content_for :page_actions do %>
-  <%= back_to_list_button(Spree::Property.model_name.human, admin_properties_path) %>
+  <%= link_to Spree.t(:properties), admin_properties_url %> /
+  <%= @property.presentation %>
```

Comment:
@priyank-gupta we had a discussion internally about it - `presentation` is more human-like in most cases and that's why we finally have chosen it over `name`